plugins/modules/mso_schema_site_external_epg.py

```diff
@@ -122,8 +171,12 @@ def main():
     if not schema_obj.get("sites"):
         mso.fail_json(msg="No site associated with template '{0}'. Associate the site with the template using mso_schema_site.".format(template))
     sites = [(s.get("siteId"), s.get("templateName")) for s in schema_obj.get("sites")]
+    sites_list = [s.get("siteId") + "/" + s.get("templateName") for s in schema_obj.get("sites")]
     if (site_id, template) not in sites:
-        mso.fail_json(msg="Provided template '{0}' does not exist. Existing templates: {1}".format(template, ", ".join(templates)))
+        mso.fail_json(
+            msg="Provided site/siteId/template '{0}/{1}/{2}' does not exist. "
```

Comment:
Do we need siteId in the error message? I assume ( which might be wrong ) that the Id would will not mean much to the user ( because the user does not provide the Id but is computed/lookedup from name ).